### DISCUSSION TITLE
Pypgstac version on stac-fastapi service doesn't include psycopg

### DISCUSSION THREAD

**Discussion by mdominguezd** [developmentseed/eoAPI#194] [Q&A]
Hi there!

I have been trying to work with the docker compose provided on the main eoAPI repo to ingest my own STAC catalog to the pgstac database. However, it seems that the version of pypgstac installed on the stac-fastapi service does not include psycopg, therefore it is not possible to connect to the database from within the container and ingest the catalog I have created.

 **Is there a reason why this package is not included there?**

For now what I do is create an override to edit that docker compose file, and it works.

**Comment by emileten** [comment]
Hi @mdominguezd, thank you for the question !

In the set of docker compose services here, the  `stac-fastapi` service wasn't meant to be used for manual ingestions.

Instead we usually manually install `pypgstac` (along with `psycopg`) in a separate virtual environment and run ingestions from there. You can find an example of this in this notebook : https://github.com/developmentseed/eoAPI/blob/main/demo/Maxar/eoAPI_Maxar_demo.ipynb. 

This should probably be documented in the getting-started docs.
